library/init/core.lean

```diff
@@ -168,10 +178,18 @@ quot.lift f _ (quot.mk a) ~~> f a
 -/
 init_quotient
 
-/-- Heterogeneous equality.
+/--
+Heterogeneous equality.
+
 It's purpose is to write down equalities between terms whose types are not definitionally equal.
 For example, given `x : vector α n` and `y : vector α (0+n)`, `x = y` doesn't typecheck but `x == y` does.
- -/
+
+If you have a goal `⊢ x == y`, 
+your first instinct should be to ask (either yourself, or on zulip)
```

Comment:
If we mention zulip, we should probably add a link.

Reply:
```suggestion
your first instinct should be to ask (either yourself, or on [zulip](https://leanprover.zulipchat.com/))
```